src/Dialog/confirmation-dialog.scss

```diff
@@ -0,0 +1,10 @@
+@import "../variables";
+
+.modal {
+    background-color: $modal-backdrop-bg;
```

Comment:
Instead of manually setting the background colour here, better change the backdrop value to `true` here (for confirmation dialogs that are not in progress): 
https://github.com/NordicSemiconductor/pc-nrfconnect-shared/blob/85722f36bbccba24dff231e9347c04f5ff695a93/src/Dialog/ConfirmationDialog.jsx#L85

Reply:
Good point, I will update.